lib/workflow/postgres/postgres.go

```diff
@@ -684,6 +706,18 @@ func (workc *PostgresWorkflowCompleter) Done(result []byte) error {
 	return errors.Wrapf(workc.tx.Commit(), "failed to mark workflow %d as complete", workc.wid)
 }
 
+func (workc *PostgresWorkflowCompleter) Fail(error error) error {
```

Comment:
```suggestion
func (workc *PostgresWorkflowCompleter) Fail(err error) error {
```

Want to avoid using `error` as a variable name I think.